backend/api/schema.py

```diff
@@ -0,0 +1,186 @@
+import graphene
+from graphene_django import DjangoObjectType
+from .models import *
+
+
+class UserType(DjangoObjectType):
+    class Meta:
+        model = User
+        fields = ('id', 'username', 'email', 'first_name', 'last_name',
+                  'is_staff', 'is_superuser', 'is_active', 'date_joined', "publickey")
```

Comment:
The quotation symbols are inconsistent here. You have been using `'`, so you should change `"publickey"` to `'publickey'` for more consistency.